nodes/widgets/ui_file_input.js

```diff
@@ -19,7 +19,7 @@ module.exports = function (RED) {
 
         // get max file size supported
         const MAX_FILESIZE_DEFAULT = 1e6
-        const maxFileSize = RED.settings.dashboard?.maxHttpBufferSize || MAX_FILESIZE_DEFAULT
+        const maxFileSize = RED.settings.dashboard?.maxHttpBufferSize || RED.settings.ui?.maxHttpBufferSize || MAX_FILESIZE_DEFAULT
```

Comment:
Is a value of `0` permitted / valid?

If yes, then perhaps we should use `??`

```suggestion
        const maxFileSize = RED.settings.dashboard?.maxHttpBufferSize ?? RED.settings.ui?.maxHttpBufferSize ?? MAX_FILESIZE_DEFAULT
```

Reply:
No `0` should not valid as this would disable all payloads